hnn_core/extracellular.py

```diff
@@ -233,19 +231,18 @@ def __init__(self, positions, *, conductivity=0.3, method='psa',
         if voltages is None:
             voltages = list()
```

Comment:
```suggestion
            voltages = np.array(list())
```
or `np.array([])` in spirit of the goal of this PR?